e2e/tests/nightly/admin/user-profile-admin.spec.ts

```diff
@@ -0,0 +1,469 @@
+
+    // Expires on is NEVER for Google 2-step verification module
+    const expiresOnCell = await accessStatusTable.getCellByValue(
+      AccessModules.GOOGLE_VERIFICATION,
+      TableColumns.EXPIRES_ON
+    );
+    const expiresOn = await expiresOnCell.getCellValue();
+    expect(expiresOn).toEqual('Never');
+
+    // Change Bypass for Google 2-step verification module
+    const statusCell = await accessStatusTable.getCellByValue(AccessModules.GOOGLE_VERIFICATION, TableColumns.STATUS);
+    const oldStatus = await statusCell.getCellValue();
+
+    const bypassSwitch = await userProfileAdminPage.getBypassSwitchForRow(AccessModules.GOOGLE_VERIFICATION);
+
+    if (oldStatus === 'Bypassed') {
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+      expect(await bypassSwitch.isOn()).toBe(true);
+      await bypassSwitch.turnOff();
+    } else {
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('No data access');
+      expect(await bypassSwitch.isOn()).toBe(false);
+      await bypassSwitch.turnOn();
+    }
+
+    const bypassSwitchCell = await accessStatusTable.getCellByValue(AccessModules.GOOGLE_VERIFICATION, 'Bypass');
+    await verifyToggleSwitchNewBackgroundColor(adminTab, bypassSwitchCell);
+
+    const saveButton = userProfileAdminPage.getSaveButton();
+    await verifyButtonEnabled(saveButton, true);
+    await saveButton.click();
+    await userProfileAdminPage.waitForLoad();
+
+    const newStatus = await statusCell.getCellValue();
+    if (oldStatus === 'Bypassed') {
+      expect(newStatus).toBe('Incomplete');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('No data access');
+    } else {
+      expect(newStatus).toBe('Bypassed');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+    }
+
+    if (newStatus === 'Incomplete') {
+      await bypassSwitch.turnOn();
+      await saveButton.click();
+      await userProfileAdminPage.waitForLoad();
+    }
+  });
+
+  test('Can bypass Controlled Tier training module', async () => {
+    const userProfileAdminPage = new UserProfileAdminPage(adminTab);
+    await userProfileAdminPage.waitForLoad();
+
+    const accessStatusTable = await userProfileAdminPage.getAccessStatusTable();
+    const expiresOnCell = await accessStatusTable.getCellByValue(AccessModules.CT_TRAINING, TableColumns.EXPIRES_ON);
+    const expiresOn = await expiresOnCell.getCellValue();
+    expect(expiresOn).toEqual('-');
+
+    // Change Bypass for Google 2-step verification module
+    const statusCell = await accessStatusTable.getCellByValue(AccessModules.CT_TRAINING, TableColumns.STATUS);
+    const oldStatus = await statusCell.getCellValue();
+
+    const bypassSwitch = await userProfileAdminPage.getBypassSwitchForRow(AccessModules.CT_TRAINING);
+    if (oldStatus === 'Bypassed') {
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
```

Comment:
why not Controlled Tier here?  that would be a more interesting test

Reply:
We need to have an institution on Test that remains in an expected state for testing.  How about something like "For e2e Testing - DO NOT CHANGE MANUALLY"